### PR TITLE
proof of concept for reporting rates facility tracking

### PR DESCRIPTION
**Pull request by czue**
@gcapalbo I wanted to see what it would look like to end-to-end the facility workflow all the way though from the warehouse to the reports. didn't end up being so bad.

I think we'll want to clean up things within this implementation but is probably an ok skeleton to build off of.

review by commit

### CODE REVIEW

File: logistics_project/apps/malawi/warehouse/runner.py [hunk 1]
```diff
@@ -100,6 +100,15 @@ def generate(self, run_record):
                 print "processing hsa %s (%s) (%s of %s)" % (hsa.name, str(hsa.id), i, count)
                 self.update_base_level_data(hsa, start, end, all_products)
 
+        if settings.ENABLE_FACILITY_WORKFLOWS:
+            print 'processing facility data'
+            facilities = SupplyPoint.objects.filter(active=True, type__code=SupplyPointCodes.FACILITY).order_by('id')
+            if self.facility_limit:
```
Comment (gcapalbo) [inline logistics_project/apps/malawi/warehouse/runner.py]: just out of curiosity - is the facility_limit one of the debugging tools used to speed up the runner or does it have another meaning?

File: logistics_project/apps/malawi/migrations/0015_auto__add_field_reportingrate_is_facility.py [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'ReportingRate.is_facility'
+        db.add_column('malawi_reportingrate', 'is_facility',
+                      self.gf('django.db.models.fields.BooleanField')(default=False),
+                      keep_default=False)
```
Comment (gcapalbo) [inline logistics_project/apps/malawi/migrations/0015_auto__add_field_reportingrate_is_facility.py]: we can discuss offline but we should check to see how many records are on this table and maybe run the migration separate from deploy if needed

**Comment by gcapalbo** [comment]
@czue looks good. is anything else needed here or is it good to merge (i see you mentioned some cleanup)?

File: logistics_project/apps/malawi/migrations/0015_auto__add_field_reportingrate_is_facility.py (hunk 2 above)
Comment (czue) [inline logistics_project/apps/malawi/migrations/0015_auto__add_field_reportingrate_is_facility.py]: sounds good. there are `402181` records according to django admin

**Comment by czue** [comment]
I think this is good to merge (though we should discuss the deploy process if that's a concern). cleanup was mostly referring to how we could clean up the split long term but I think for the scope of this PR it's fine.

**Comment by gcapalbo** [comment]
ok, merging (i think normal deploy will be fine)

**Pull request merged**